common/org.testeditor.dsl.common.ui.tests/.settings/org.eclipse.xtend.core.Xtend.prefs

```diff
@@ -0,0 +1,33 @@
+ValidatorConfiguration.is_project_specific=true
```

Comment:
Unfortunately we need to set project specific settings here as the Xtend editor does not properly support `@SupressWarning` - we use `EditorDocumentChange` which is marked as restricted.